packages/arb-bridge-peripherals/test/customGateway.e2e.ts

```diff
@@ -307,7 +307,7 @@ describe('Bridge peripherals end-to-end custom gateway', () => {
     )
   })
 
-  it('should withdraw tokens when minted in L2', async function () {
+  it.skip('should withdraw tokens when minted in L2', async function () {
```

Comment:
lets update this test so it expects that the L1 side reverts when trying to withdraw tokens minted in L2